eth/tracers/firehose.go

```diff
@@ -157,6 +160,7 @@ type Firehose struct {
 	// here. If not set in the config, then we inspect `OnBlockchainInit` the chain config to determine
 	// if it's a network for which we must reproduce the legacy bugs.
 	applyBackwardCompatibility *bool
+	concurrentBlockFlushing    bool
```

Comment:
@sduchesneau Any idea about a better name here for the config, `block flushing`, don't feel it's super clear.